metadata-db-app/src/cmr/metadata_db/data/memory_db.clj

```diff
@@ -309,15 +310,10 @@
 (defn generate-concept-id
   [db concept]
   (let [{:keys [concept-type provider-id]} concept
-        concept-sub-type (get concept :concept-sub-type)
         num (swap! (:next-id-atom db) inc)]
-    (if (some? concept-sub-type)
-      (cc/build-generic-concept-id {:concept-type concept-sub-type
```

Comment:
I didn't see build-generic-concept-id being deleted from cmr.common.concepts.  Is it being used somewhere else?

Reply:
removed